routes/api/games.js

```diff
@@ -0,0 +1,33 @@
+const express = require('express');
+const router = express.Router();
+const mongoose = require('mongoose');
+const passport = require('passport');
+const Game = require('../../models/Game');
+// const getFirstWord method required to fetch first word from database
+// getNextWord
+
+// Frontend sends userId in post request
+router.post('/game', (req, res) => {
+  //is this the route we want?
+  const userId = req.userId;
+  const newGame = new Game({
+    user: userId,
+    wordsSent: [], // send result of getFirstWord
+    wordsGuessed: [],
+    score: 0,
```

Comment:
Add a timer/expiration datetime (Moment makes it really easy to get the datetime string for 5 minutes in the future)